scripts/tools/silabs/FactoryDataProvider.py

```diff
@@ -285,6 +288,18 @@ def create_nvm3injected_image(self):
             productUrlByteArray = bytes(self._args.product_url, 'utf-8').hex()
             cmd.extend(["--object", self.PRODUCT_URL_NVM3_KEY + str(productUrlByteArray)])
 
+        if self._args.ca_cert:
+            ca_certArray = bytes(self._args.ca_cert, 'utf-8').hex()
```

Comment:
there is no check done on those new arguments. Is there a size limit or something you'd want to validate before storing it? Those checks would usually done in
`__init__`

Reply:
@Thirsrin an answer to this question would be appreciated

Reply:
he did add some size limit checks to the __init__ I don't know if there is other elements that should be validated